### PR TITLE
Add support for Rocky Linux

### LINKED ISSUE #471: Installation on Rocky Linux
Rocky Linux 8 is very similar to CentOS 8 but only CentOS 8 is supported by Sysbox. Is there a way to compile Sysbox on Rocky Linux (or other similar distributions)?

### PR DESCRIPTION
**Pull request by roussidis**
- Add the required Dockerfile to build sysbox in Rocky Linux OS
- Add Rocky Linux as a supported OS wherever that is needed
- Add missing almalinux as a supported in scr/sysbox file

Closes #471

Signed-off-by: Christos Roussidis <xristos.roussidis@gmail.com>

### CODE REVIEW

File: tests/Dockerfile.rocky-8 [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+#
+# This Dockerfile is based on a similar Dockerfile in the OCI runc
+# github repo, but adapted to sysbox testing.
+#
+# Instructions:
+#
+# docker build -t sysbox-test .
+#
+
+FROM rockylinux
```
Comment (rodnymolina) [inline tests/Dockerfile.rocky-8:22]: Please set an explicit tag here (e.g. "rockylinux:8.5") to ensure that we don't break anything when bringing a new "latest" release.

**Review by rodnymolina — CHANGES_REQUESTED** [review]
Minor change request. Looks good other than that.

The issues being observed (reported through slack) are not a consequence of these changes, and will need to be addressed separately.

File: tests/Dockerfile.rocky-8 (hunk 1 above)
Comment (roussidis) [reply tests/Dockerfile.rocky-8:22]: Makes sense to change that. Should we do the change in the tests/Dockerfile.almalinux-8 file accordingly ?

Comment (rodnymolina) [reply tests/Dockerfile.rocky-8:22]: Oh we missed that. Yes please, do the same in the almalinux file too. Thanks!

**Pull request merged**